includes/general.php

```diff
@@ -964,6 +970,12 @@ function pods_shortcode_run( $tags, $content = null ) {
 		$params = array();
 
 		if ( ! defined( 'PODS_DISABLE_SHORTCODE_SQL' ) || ! PODS_DISABLE_SHORTCODE_SQL ) {
+			$evaluate_tags_args = array(
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: ece3b20d843d6bfeace0ea023d96db93</sub>